examples/base/package-lock.json

```diff
@@ -0,0 +1,1691 @@
+{
```

Comment:
Please do not include shrinkwrap in project. Product will get those in productization process.
See productization ticket for more details.